js_modules/dagster-ui/packages/ui-core/src/assets/AssetsCatalogTable.tsx

```diff
@@ -225,7 +225,7 @@ export const AssetsCatalogTable = ({
 
   const refreshState = useRefreshAtInterval({
     refresh: query,
-    intervalMs: FIFTEEN_SECONDS,
+    intervalMs: 4 * FIFTEEN_SECONDS,
```

Comment:
This seems fairly impactful, are we ok with backing this off so much? It should already be slowing down if `refresh` takes a while, there's some fanciness inside this hook

Reply:
This still fires immediately if they leave tabs or come back or hit the refresh button in the top right corner.